ballerina/tests/producer_client_tests.bal

```diff
@@ -180,7 +180,7 @@ function producerGetTopicPartitionsErrorTest() returns error? {
     check producer->close();
 }
 
-@test:Config {enable: true}
+@test:Config {enable: false}
```

Comment:
We can create a issue for this

Reply:
created https://github.com/ballerina-platform/ballerina-library/issues/7372